src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useMarket, Market } from 'components/utils/CurrentLocale'
+import { DanishDetails } from './DanishDetails'
+import { NorwegianDetails } from './NorwegianDetails'
+import { SwedishDetails } from './SwedishDetails'
+import { DetailsProps } from './types'
+
+export const Details = ({
+  fieldSchema,
+  formikProps,
+  offerQuote,
+  textKeys,
```

Comment:
Hm, i didn't see this before but any partucular reason for passing the text keys as a property? If not i think we should try and use the `useTextKeys` hook inside the components that need them instead. Feels a bit messy to pass it as a prop in some places and use the hook in other.